<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblbackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../MenuPy.gif</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHangman">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>310</y>
      <width>348</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72, 0%);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJokenpo">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>442</y>
      <width>348</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72, 0%);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMemento">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>310</y>
      <width>348</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72, 0%);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHash">
    <property name="geometry">
     <rect>
      <x>836</x>
      <y>442</y>
      <width>348</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72, 0%);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
